TribalWars/src/Infrastructure/Repository/TribeRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using ApplicationCore.Entities;
+using ApplicationCore.Interfaces.Repository;
+using Infrastructure.Data;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Repository
+{
+    public class TribeRepository : BaseRepository, ITribeRepository
+    {
+        public TribeRepository(ApplicationDbContext context) : base(context)
+        {
+        }
+
+        public async Task<int> CreateTribe(Tribe tribe)
+        {
+            await Context.Tribes.AddAsync(tribe);
+            await Context.SaveChangesAsync();
+            return tribe.Id;
+        }
+        public async Task AddPlayerToTribe(TribePlayer player)
+        {
+            await Context.TribePlayers.AddAsync(player);
+            await Context.SaveChangesAsync();
+        }
+
+        public async Task<Tribe> SelectedTribe(string tribeName)
```

Comment:
Imho SelectedTribe is a little bit misplaced, because this method is going to query tribe by name.